src/ceph-volume/ceph_volume/devices/rawbluestore/activate.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import print_function
+import argparse
+import logging
+import os
+from textwrap import dedent
+from ceph_volume import process, conf, decorators, terminal, __release__, configuration
+from ceph_volume.util import system, disk
+from ceph_volume.util import prepare as prepare_utils
+from ceph_volume.util import encryption as encryption_utils
+from ceph_volume.systemd import systemctl
+from ceph_volume.api import lvm as api
+from .list import direct_report
+
+
+logger = logging.getLogger(__name__)
+
+def activate_bluestore(meta, no_systemd=False):
+    # find the osd
+    osd_id = meta['osd_id']
+    osd_fsid = meta['osd_uuid']
+
+    # mount on tmpfs the osd directory
+    osd_path = '/var/lib/ceph/osd/%s-%s' % (conf.cluster, osd_id)
+    if not system.path_is_mounted(osd_path):
```

Comment:
The path might exist and the device might not be mounted, what is the concern about checking mounts?

Reply:
I think if we remove the mount check this stops working the way things normally do in non-containerized context: after a reboot the mount point will generally still exist in /var/lib/ceph/osd, but we still want tmpfs in that case.

For containers (including rook), this will be a (bind) mount point and we'll just use it.

Reply:
> I think if we remove the mount check this stops working the way things normally do in non-containerized context: after a reboot the mount point will generally still exist in /var/lib/ceph/osd, but we still want tmpfs in that case.
> 

I guess we want is this directory not to be empty right?

> For containers (including rook), this will be a (bind) mount point and we'll just use it.

Unfortunately, this is not what I observed recently, c-v mounts it anyway.

Reply:
I can add a --no-tmpfs option then...